code/src/components/SurveyForm.js

```diff
@@ -0,0 +1,63 @@
+import React, {useState} from 'react'
+import Start from './Start.js'
+import Age from './Age.js'
+import Purpose from './Purpose.js'
+import Expectations from './Expectations.js'
+import Summary from './Summary.js'
+
+const SurveyForm = () => {
+const [ageGroup, setAgeGroup] = useState('')
+const [options, setOptions] = useState ('choose below')
+const [expectations, setExpectations] = useState('')
+const [slide, setSlide] = useState(1)
+
+const nextSlide = () => {
+    setSlide(slide + 1)
+}
+const backSlide = () => {
+    setSlide(slide - 1)
+}
+//Rendering all components 
+    return (
+      <div>
+        <form>
+            {slide === 1 && (
+            <Start nextSlide={nextSlide} />
+            )}
+            {slide === 2 && (
+            <Age 
+            ageGroup={ageGroup} 
```

Comment:
Such a clean and unmessy code but maybe think of tabbing in the attributes so they don't all start from the same line. For example 
```
`<Age 
    ageGroup={ageGroup}
    setAgeGroup={setAgeGroup}`
   />
```